spec/api_auth_spec.rb

```diff
@@ -94,6 +112,24 @@ def hmac(secret_key, request)
       request['date'] = "٢٠١٤-٠٩-٠٨ ١٦:٣١:١٤ +٠٣٠٠"
       expect(ApiAuth.authentic?(request, "123")).to eq false
     end
+
+    context "canonical string contains the http_method" do
```

Comment:
I guess I just didn't see a test for that case anywhere else. Especially given that preventing that sort of request was the point of the change.